### PR TITLE
Feature/delete user

### PR DESCRIPTION
**Pull request by Lloyd-Brombach-WSU**
I think the feature is complete. Added a button on index.html to navigate to manageUsers.html. Button appears when logged in as admin)

### CODE REVIEW

File: src/test/java/org/redcrosswarriors/controllerservice/TestFeedbackControllerService.java [hunk 1]
```diff
@@ -1,72 +1,74 @@
-package org.redcrosswarriors.controllerservice;
```
Comment (jacobboesch) [inline src/test/java/org/redcrosswarriors/controllerservice/TestFeedbackControllerService.java:1]: I'm not really sure why this file shows up on the pull request seeing that you didn't change anything here. But I guess it doesn't matter.

**Review by jacobboesch — CHANGES_REQUESTED** [review]
Everything looks good. Except I can't seem to find a link to the manage users page. 
Could you please add this to the navigation if the user is an admin?

File: src/main/resources/static/index.html [hunk 2]
```diff
@@ -16,16 +16,18 @@
         <!--signup will be a button next to login that turns into a signout after a user is logged in-->
 
         <div class="ui right floated header">
+            <button class="ui left floated green inverted button" id="manageUsersNavButton" style="display: none;"
```
Comment (jacobboesch) [inline src/main/resources/static/index.html]: Could you possibly change the color of the button so that it doesn't blend in with the background.

**Review by jacobboesch — CHANGES_REQUESTED** [review]
Button works but it blends in with the background. If you could change the color of the button to something else that would be great.

File: src/test/java/org/redcrosswarriors/controllerservice/TestFeedbackControllerService.java (hunk 1 above)
Comment (Lloyd-Brombach-WSU) [reply src/test/java/org/redcrosswarriors/controllerservice/TestFeedbackControllerService.java:1]: possible I errantly added a space while studying

File: src/main/resources/static/index.html (hunk 2 above)
Comment (Lloyd-Brombach-WSU) [reply src/main/resources/static/index.html]: Done deal. I have no idea what happened to the button in my initial pull request.. I must have lost it when I did a last pull and merge from master before I pull requested. It's back.

**Review by jacobboesch — APPROVED** [review]

**Review by TheKPham — APPROVED** [review]

**Review by tia-gijo — APPROVED** [review]

**Review by jacobboesch — APPROVED** [review]

File: src/main/resources/templates/manageUsers.html [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+<div class="ui fluid container">
+    <div class="content">
+        <div class="transparent-bg">
+            <img src="Blood-Donation.jpg" id="bdimg" style="width:100%; background-size: cover;">
+        </div>
+        <div class="ui left aligned container">
+            <div class="centered">
+                <div class="userListContainer" id="userListContainer">
+                    <div class="ui placeholder segment" id="fogCover" style="background: rgba(0,0,0,.7); max-width: 753px; max-height: 349px; overflow-y: auto;">
+                        <h2>Feedback</h2>
```
Comment (Kyle-WSU) [inline src/main/resources/templates/manageUsers.html]: rename to something appropriate to the page

**Review by Kyle-WSU — CHANGES_REQUESTED** [review]

**Review by Kyle-WSU — CHANGES_REQUESTED** [review]
Just rename the header file i mentioned and ill approve the request

**Review by Kyle-WSU — APPROVED** [review]

File: src/main/resources/templates/manageUsers.html (hunk 3 above)
Comment (Lloyd-Brombach-WSU) [reply src/main/resources/templates/manageUsers.html]: done

**Pull request merged**